assignment/assignment1/server/src/routes/questions.ts

```diff
@@ -8,6 +8,15 @@ const router = express.Router();
 router.get("/", verifyToken, getAllQuestions);
```

Comment:
Is this route a repeat of the route registered in index.ts?

Reply:
yea. I removed the one at index.ts